test/sai_test/sai_test_base.py

```diff
@@ -214,6 +217,18 @@ class T0TestBase(ThriftInterfaceDataPlane):
         self.local_server_mac_list for all the local server mac
     """
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.dut = Dut()
+        self.num_device_each_group = 99
+        self.server_group0, self.server_group1, self.server_group2, self.server_group3 = [], [], [], []
+        self.t0_group1, self.t0_group2, self.t0_group3 = [], [], []
+        self.t1_group1, self.t1_group2, self.t1_group3 = [], [], []
+        self.create_device()
+        self.lag1_neighbor = self.t1_group1[0]
```

Comment:
Is it better to put the assignment of these two variables in lag_config.py after create lag?